actions/push-to-gar-docker/action.yaml

```
@@ -170,3 +170,9 @@ runs:
         platforms: ${{ inputs.platforms }}
         ssh: ${{ inputs.ssh }}
         build-contexts: ${{ inputs.build-contexts }}
+
+    - name: Cleanup checkout directory
+      if: always()
+      shell: bash
+      run: |
+        rm -rf _shared-workflows-push-to-gar
```

since this is going to run all the time it might be an idea to be defensive

```suggestion
      run: |
        # Check that the directory looks OK before removing it
        if ! [ -d "_shared-workflows-push-to-gar/.git" ]; then
          echo "::warning Not removing shared workflows directory: doesn't look like a git repository"
          exit 0
        fi

        rm -rf _shared-workflows-push-to-gar
```